ERROR - 2021-10-26 01:35:42 --> Table 'ecommerce.users' doesn't exist
CRITICAL - 2021-10-26 01:35:42 --> Table 'ecommerce.users' doesn't exist
#0 C:\xampp\htdocs\Activity_11\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(280): mysqli->query('SELECT 1\nFROM `...')
#1 C:\xampp\htdocs\Activity_11\vendor\codeigniter4\framework\system\Database\BaseConnection.php(663): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1\nFROM `...')
#2 C:\xampp\htdocs\Activity_11\vendor\codeigniter4\framework\system\Database\BaseConnection.php(591): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1\nFROM `...')
#3 C:\xampp\htdocs\Activity_11\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1428): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#4 C:\xampp\htdocs\Activity_11\vendor\codeigniter4\framework\system\Validation\Rules.php(161): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\Activity_11\vendor\codeigniter4\framework\system\Validation\Validation.php(277): CodeIgniter\Validation\Rules->is_unique('test@gmail.com', 'email', Array, NULL)
#6 C:\xampp\htdocs\Activity_11\vendor\codeigniter4\framework\system\Validation\Validation.php(151): CodeIgniter\Validation\Validation->processRules('email', 'email', 'test@gmail.com', Array, Array)
#7 C:\xampp\htdocs\Activity_11\vendor\codeigniter4\framework\system\Controller.php(160): CodeIgniter\Validation\Validation->run()
#8 C:\xampp\htdocs\Activity_11\app\Controllers\Register.php(22): CodeIgniter\Controller->validate(Array)
#9 C:\xampp\htdocs\Activity_11\vendor\codeigniter4\framework\system\CodeIgniter.php(802): App\Controllers\Register->new_user()
#10 C:\xampp\htdocs\Activity_11\vendor\codeigniter4\framework\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#11 C:\xampp\htdocs\Activity_11\vendor\codeigniter4\framework\system\CodeIgniter.php(317): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\Activity_11\public\index.php(37): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\Activity_11\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#14 {main}
ERROR - 2021-10-26 01:41:40 --> Cannot add or update a child row: a foreign key constraint fails (`ecommerce`.`watumizi`, CONSTRAINT `watumizi_ibfk_1` FOREIGN KEY (`role`) REFERENCES `tbl_roles` (`role_id`))
CRITICAL - 2021-10-26 01:41:40 --> Cannot add or update a child row: a foreign key constraint fails (`ecommerce`.`watumizi`, CONSTRAINT `watumizi_ibfk_1` FOREIGN KEY (`role`) REFERENCES `tbl_roles` (`role_id`))
#0 C:\xampp\htdocs\Activity_11\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(280): mysqli->query('INSERT INTO `wa...')
#1 C:\xampp\htdocs\Activity_11\vendor\codeigniter4\framework\system\Database\BaseConnection.php(663): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `wa...')
#2 C:\xampp\htdocs\Activity_11\vendor\codeigniter4\framework\system\Database\BaseConnection.php(591): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `wa...')
#3 C:\xampp\htdocs\Activity_11\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('INSERT INTO `wa...', Array, false)
#4 C:\xampp\htdocs\Activity_11\vendor\codeigniter4\framework\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\Activity_11\vendor\codeigniter4\framework\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\Activity_11\vendor\codeigniter4\framework\system\Model.php(620): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\Activity_11\app\Controllers\Register.php(80): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\Activity_11\vendor\codeigniter4\framework\system\CodeIgniter.php(802): App\Controllers\Register->new_user()
#9 C:\xampp\htdocs\Activity_11\vendor\codeigniter4\framework\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#10 C:\xampp\htdocs\Activity_11\vendor\codeigniter4\framework\system\CodeIgniter.php(317): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\Activity_11\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\Activity_11\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#13 {main}
CRITICAL - 2021-10-26 04:15:01 --> Trying to access array offset on value of type null
#0 C:\xampp\htdocs\Activity_11\app\Controllers\Register.php(119): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Trying to acces...', 'C:\\xampp\\htdocs...', 119)
#1 C:\xampp\htdocs\Activity_11\vendor\codeigniter4\framework\system\CodeIgniter.php(802): App\Controllers\Register->auth()
#2 C:\xampp\htdocs\Activity_11\vendor\codeigniter4\framework\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#3 C:\xampp\htdocs\Activity_11\vendor\codeigniter4\framework\system\CodeIgniter.php(317): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\Activity_11\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\Activity_11\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#6 {main}
